components/intake.py

```diff
@@ -0,0 +1,24 @@
+from magicbot import tunable
+import wpilib
+
+
+class Intake:
+    shooter_motor: wpilib.Talon
```

Comment:
plz rename adequately for your variables intake isn't shooting...

Reply:
agreed, this one should be simply named `motor`; you already know this is the intake